lib/mutant/integration/minitest.rb

```diff
@@ -95,6 +95,8 @@ def call(tests)
           break unless test.call(reporter)
         end
 
+        reporter.report
```

Comment:
The minitest integration itself does not have a full mutation covering test right now. It only has integration tests.

This is for the reason mutant (at this point) is not prepared to load multiple integrations the same time, and as mutant itself requires rspec on itself this is a bit of a tie.

This situation should improve in the future.